mithril-ui/App.js

```diff
@@ -0,0 +1,18 @@
+import {Menu} from './menu/Menu.js'
+import {appState, menuState} from './store/state.js';
+
+export const App = (initialVnode) => {
+  return {
+    oninit: (vnode) => {
+      console.log("Hello Mithrel", appState)
```

Comment:
No need to log this and spam the console for everyone. Can you comment it out?

On your local dev env you can uncomment but simply not commit that.